build.yaml

```diff
@@ -3074,15 +3075,11 @@ steps:
       hl.utils.range_table(10).collect()
       EOF
 
-      BATCH_ID=$(hailctl batch submit simple_hail.py --name=test-hailctl-batch-submit --files=foo -o json | jq '.id')
+      BATCH_ID=$(hailctl batch submit -v simple_hail.py:/ -v foo/:/foo/ --workdir / --name=test-hailctl-batch-submit -o json python3 simple_hail.py | jq '.id')
       STATUS=$(hailctl batch wait -o json $BATCH_ID)
-      STATE=$(echo $STATUS | jq -jr '.state')
-      if [ "$STATE" == "success" ]; then
-          exit 0;
-      else
-          echo $STATUS;
-          exit 1;
-      fi
+      hailctl batch job $BATCH_ID 1          # print job status
+      hailctl batch log -o json $BATCH_ID 1  # print the logs
+      test $(jq -jr '.state' <<< "$STATUS") = "success"
```

Comment:
With `set -e`, we'll exit if the test command fails, no need for `if ...; exit 1; fi`.